src/MDMTransition.h

```diff
@@ -37,6 +37,17 @@ NS_SWIFT_NAME(Transition)
 @end
 
 /**
+ Transition with custom duration overrides the default transition duration
+ */
+NS_SWIFT_NAME(TransitionWithCustomDuration)
+@protocol TransitionWithCustomDuration
+/**
+ Return the duration for the transition
+ */
+- (NSTimeInterval)transitionDuration;
```

Comment:
We will likely want to provide this method with the context in case the transition implementation wants to provide different timings for forward/backward transitions.

`transitionDurationWithContext:(id<...>)context` should be sufficient.